pages/wg/rse-gln.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: RSE Group Leaders' Network
+subtitle:
+menubar: working-groups
+permalink: /wg/rse-gln/
+set_last_modified: true
+---
+
+# Overview
+
+The RSE Group Leaders’ Network (RSE-GLN) is a group of individuals leading RSE teams who meet to support their peers with the challenges of leading RSE groups.  
+
+# Membership
+
+The primary membership is for those who currently lead a team and have, or are actively in the process of hiring, at least one direct report; additional members may be invited, such as those who have been in leadership positions in the past.  
+
+The RSE-GLN also seeks to support aspiring RSE Group Leaders through a separate mentorship and development network.  The Aspiring RSE-GLN is open to those looking to build RSE teams at their institution or move into RSE group leadership roles.  There are no formal requirements, but those with the experience and qualifications to take on leadership roles will likely benefit more than early career RSEs.  
+
+In both cases, membership is open to [US-RSE members](/join/).  As with other US-RSE activities, the RSE-GLN will be US-focused, but those from other countries are welcome.  
+
+To join the RSE-GLN, fill out the [RSE-GLN membership form](https://docs.google.com/forms/d/e/1FAIpQLScMxqpp8q4xB8WQBjy8bh_MtUGWn_JUSbeb2buefgHZYJJQwg/viewform?usp=sf_link) and someone will be in touch.
```

Comment:
it's not clear if this is open to everyone with some people encouraged, or if this is only open to certain people - it would be better (less confusing) if this was more clear, I think.

Reply:
The aspiring group is open to anyone.  The main network is open to people who meet the criteria of having a direct report or at least being in the process of hiring one (self-identified) or are specifically invited.  The membership form asks what category someone is in, so that we can put them in the right part of the group.  Does that work, or is there something else you think would help?

Reply:
maybe change the part above to say

In both cases, membership is open to all [US-RSE members](/join/). As with other US-RSE activities, the RSE-GLN will be US-focused, but those from other countries are welcome.